### PR TITLE
adds the translation for the ukrainian language

### PR DESCRIPTION
**Pull request by dbilousov**
#### What kind of PR is this?:
/kind feature

#### What this PR does / why we need it:
It adds the translation for the Ukrainian language. I think the DefiChain needs more attention in the eastern Europe. Usually eastern nations like Ukrainian have bad English knowledge. This change should attract those people to use DefiChain.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1 @@
-{
-  "workbench.colorCustomizations": {
-    "titleBar.activeBackground": "#f069c5",
-    "titleBar.activeForeground": "#000000"
```
Comment (thedoublejay) [inline .vscode/settings.json:4]: why remove this?

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (dbilousov) [reply .vscode/settings.json:4]: You are right. It looks like my VS didn't like the color. Will change it back.

**Comment by dbilousov** [comment]
What can I do to push the review forward? Do you guys need an additional reviewer who speaks Ukrainian?

**Comment by thedoublejay** [comment]
Hi, should be ok @dbilousov. Thank you! If you can, please create one also for https://github.com/DeFiCh/wallet. I believe a lot of users are now using it.

**Comment by dbilousov** [comment]
Hi @thedoublejay 
thank you! Yes, I am going to translate into the Ukrainian the light wallet too 👍

**Pull request merged**